core/build.gradle

```diff
@@ -26,31 +26,15 @@ apply plugin: 'idea'
 
```

Comment:
I think merge should remove the line above
```
apply plugin: 'idea'
```

Reply:
👍